### ISSUE TITLE
Add option to hide Default section in subreddit picker

### ISSUE THREAD

**Issue by intrapt** [laurencedawson/sync-for-reddit#1202]
Currently, all sections except the Default one can be hidden.

![IMG_20220504_205325](https://user-images.githubusercontent.com/46963987/166815429-a71a74ac-1b47-44ad-ae52-06dc4822769f.jpeg)

**Comment by laurencedawson** [comment]
Adding a discussion tag as I really don't think this should be an open but I'll leave it open for now.

**Comment by greenskye** [comment]
I was able to do this previously: 
![Screenshot_20220504-152026_Sync Pro](https://user-images.githubusercontent.com/12568635/166819330-4b16acbd-fa43-4387-8978-a6d7370e8a37.jpg)

Not sure why it was removed, but I preferred it the other way. I prefer to keep a handful of subreddits at the top (as shown in the screenshot) where the defaults are just another favorite. Giving them special status that can't be removed is mildly annoying. Not the biggest deal, but I'd prefer this change reverted.
